app/view_models/workarea/storefront/search_autocomplete_view_model.rb

```diff
@@ -15,15 +24,19 @@ def response
       end
 
       def searches
-        @searches ||= Metrics::SearchByWeek.autocomplete(
+        return results unless results.blank?
+
+        Workarea::Insights::TrendingSearches.current.results.map do |v|
+          v['query_id']
+        end
+      end
+
+      def results
```

Comment:
Why switch this with `searches`? It would read nicer in the view if it were something like
```
  - if @autocomplete.searches.blank?
    - @autocomplete.trending_searches.each do |search|
```

Which the language then describes the functionality which is to fall back to trending searches if the autocompleted searches are empty.